File: iOS/DataPool/Sources/DataPool/Domain/PoolMemberResponse.swift
struct PoolMemberResponse: Codable {
    let gamblerId: String
    let gamblerUsername: String
    let gamblerEmail: String
    let isOwner: Bool
}

extension PoolMemberResponse {
    func toPoolMember() -> PoolMember {
        return PoolMember(
            gamblerId: self.gamblerId,
            gamblerUsername: self.gamblerUsername,
            gamblerEmail: self.gamblerEmail,
            isOwner: self.isOwner
        )
    }
}

File: Android/data/pool/src/main/java/com/felipearpa/tyche/data/pool/domain/PoolMemberResponse.kt
package com.felipearpa.tyche.data.pool.domain

import kotlinx.serialization.Serializable

@Serializable
internal data class PoolMemberResponse(
    val gamblerId: String,
    val gamblerUsername: String,
    val gamblerEmail: String,
    val isOwner: Boolean,
)
